.github/codecov.yml

```diff
@@ -0,0 +1,2 @@
+comment:
+  require_changes: true
```

Comment:
They say to set it to `false` in the text, and then write `true` in the code snippet: it is definitely confusing.

You can see it in EKO here:
https://github.com/NNPDF/eko/blob/e3466fae35fd245c7e1a99158d8582f590276555/.github/codecov.yml#L30-L33

`no` is equivalent to `false`, but `false` is better because of the [Norway problem](https://hitchdev.com/strictyaml/why/implicit-typing-removed/)

Reply:
Ok, so I need to set it `false`.
Where should I write `true` instead (which code snippet)?

Reply:
> Where should I write `true` instead (which code snippet)?

Nowhere: I was just saying that in the docs they first say to set it to `false`, and then present a code snippet with `true`. Their intent is to show you "look, this is `true` and you should turn to `false`", but the most intuitive thing would be to show as it should be (such that you can copy), not how it is before you will change.

Reply:
Oh I see. Indeed it is very confusing.